README.md

```diff
@@ -9,23 +9,21 @@
 
 [Getting Started](https://www.mackelab.org/sbi/tutorial/00_getting_started/) | [Documentation](https://www.mackelab.org/sbi/)
 
-`sbi` is a PyTorch package for simulation-based inference. Simulation-based inference is  
-the process of finding parameters of a simulator from observations.
+`sbi` is a PyTorch package for simulation-based inference. Simulation-based inference is the process of finding parameters of a simulator from observations.
 
 `sbi` takes a Bayesian approach and returns a full posterior distribution
-over the parameters, conditional on the observations. This posterior can be amortized (i.e.
-useful for any observation) or focused (i.e. tailored to a particular observation), with different
-computational trade-offs.
+over the parameters, conditional on the observations. This posterior can be amortized (i.e. once trained, it can be
+applied to any observation) or focused (i.e. tailored to a particular observation), with different computational trade-offs.
 
-`sbi` offers a simple interface for one-line posterior inference.
+`sbi` offers a simple interface for one-line posterior inference:
 
 ```python
 from sbi.inference import infer
 # import your simulator, define your prior over the parameters
-parameter_posterior = infer(simulator, prior, method='SNPE', num_simulations=100)
+parameter_posterior = infer(simulator, prior, method='NPE', num_simulations=100)
 ```
 
-See below for the available methods of inference, `SNPE`, `SNRE` and `SNLE`.
+`sbi` implements a variety of inference methods, including amortized and sequential algorithms. See below for an overview.
```

Comment:
Just a nickpick upon which I stumbled as a newbie. `amortized` and `sequential` fall from heaven here. I would encourage you to give a layman's description for them. This helps beginners (especially those from domain sciences) which are not used to this terminology.

For example:
```suggestion
`sbi` implements a variety of inference methods, including amortized and sequential algorithms. Amortized algorithms are suitable for simulators which are cheap to run and provide large datasets of simulations easily. Sequential algorithms are useful in cases where only a handful simulations can be obtained. See below for an overview of our algorithms.
```

Reply:
Thanks for chipping in here Peter! 
I agree, those terms should be explained shortly. But I think do have a short explanation at the beginning of the README.md. I will consider extending it accordingly.